src/macaron/slsa_analyzer/checks/check_result.py

```diff
@@ -47,18 +69,59 @@ class CheckInfo:
 class CheckResultData:
     """This class stores the result of a check."""
 
-    #: List of justifications describing the reasons for the check result.
-    #: If an element in the justification is a string,
-    #: it will be displayed as a string, if it is a mapping,
-    #: the value will be rendered as a hyperlink in the html report.
-    justification: Justification
-
     #: List of result tables produced by the check.
-    result_tables: ResultTables
+    result_tables: list[CheckFacts]
```

Comment:
This may be out-of-scope of this PR, but I think this field should be `result_facts` instead of `result_tables`, i.e. the word "tables" here is not appropriate. We may create an issue if you think this should be changed in another PR.